src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java

```diff
@@ -136,6 +141,191 @@ private void cleanupEnumNamingConflicts() {
     }
   }
 
+  private boolean applyLroConfiguration() {
+    //
+    // 1. Set `operation_field` annotations (Operation fields essential for LRO).
+    //
+    Message operation = allMessages.get("Operation");
+    if (operation == null) {
+      // This service does not have LRO - do nothing;
+      return false;
+    }
+
+    Map<ProtoOptionValues, Field> opFields = new HashMap<>();
+    for (Field field : operation.getFieldsWithNumbers().values()) {
+      String fieldName = field.getName();
+      if ("name".equals(fieldName)
+          || ("id".equals(fieldName) && !opFields.containsKey(ProtoOptionValues.NAME))) {
+        opFields.put(ProtoOptionValues.NAME, field);
+      } else if ("done".equals(fieldName)
+          || ("status".equals(fieldName) && !opFields.containsKey(ProtoOptionValues.STATUS))) {
+        opFields.put(ProtoOptionValues.STATUS, field);
+      } else if (fieldName.contains("error")) {
+        if (fieldName.contains("code")) {
```

Comment:
Curious that `fieldName` is equal to `error_code`, any other case error code naming become `error_code_xxx` or `error_xxx_code`? 
If no, is that better to fix the name check as field name equal to 'error_code`?
Same for error_essage.

Reply:
It is all guesses, unfortunatelly =(. It is not pretty either way. With the current logic I tried to keep it more generic, than it would be if we were checking for `error_code` explicitly. I think this logic is likely get updated once we hit more diregapic APIs. We could do error_code check too. Unless you have a strong opinion about it I would like keeping it as is basicaly becuase this is how I got it all tested end to end.

Reply:
Thanks for explanation. Make sense to me now.